src/components/PersonLink.tsx

```diff
@@ -0,0 +1,62 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const params = useParams();
+  const { sex, name, born, died, fatherName, motherName, slug } = person;
+
+  const parents = people.filter(
+    p => p.name === motherName || p.name === fatherName,
+  );
+
+  const mother = parents.find(p => p.name === motherName) || null;
+  const father = parents.find(p => p.name === fatherName) || null;
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': params?.slug === slug })}
+    >
+      <td>
+        <Link
+          className={cn({ 'has-text-danger': sex === 'f' })}
```

Comment:
The logic to apply the 'has-text-danger' class based on the `sex` property should be consistent. In the `mother` link (line 40), the class is applied unconditionally. Consider making the class application consistent across similar elements.